content/pages/sprint/index.rst

```diff
@@ -10,6 +10,12 @@ Sciwork hosts coding sprints for open-source projects for scientific,
 numerical, and engineering applications.  This page lists the sprints that
 hosted by sciwork.
 
+Scisprint 2022
+==============
+* `Scisprint 2022 Early January: Python ZH TW Doc
```

Comment:
Could you add a blankline between the section title, just like how it was for 2021?